packages/next/test/utils.ts

```diff
@@ -21,6 +22,21 @@ export function toHeaders(headers: IncomingMessage['headers'] = {}) {
   return obj;
 }
 
+export async function normalizeReactVersion(projectPath: string) {
+  const pkgJson = await fs.readJSON(path.join(projectPath, 'package.json'));
+
+  if (pkgJson.dependencies?.['next'] === 'canary') {
+    const res = await fetch('https://unpkg.com/next@canary/package.json');
```

Comment:
`npm view next@canary --json peerDependencies` also works and has the benefit of not relying on another registry (here unpkg). That's what we're using on Front syncs